content/articles/object-based-image-analysis-using-matlab/index.md

```diff
@@ -0,0 +1,241 @@
+
+### Introduction
+Object-based image analysis is the processing of an image based on the classification of its pixels to get more helpful information on the image. For example, such data can be based on height, edges, and boundaries. Matlab provides an interactive environment for object-based image analysis by executing functions used in analysis or inbuilt apps for image processing.
+
```

Comment:
what do you mean by 'For example, such data......". Give a precise description of what you're covering.